pkg/trace/api/api.go

```diff
@@ -371,6 +371,9 @@ const (
 	// This value is used for metrics and could be used in the future to adjust priority rates.
 	headerDroppedP0Spans = "Datadog-Client-Dropped-P0-Spans"
 
+	// headerRatesPayloadVersion TODO
+	headerRatesPayloadVersion = "Datadog-Rates-Payload-Version"
```

Comment:
There is `ETag` for a similar purpose. But, it looks like `ETag` is used when requesting a resource. But, in our case, tracing libraries submit their data, and receive the new sampling rates in exchange.

Reply:
The spec says `X-Rates-Payload-Version`

Reply:
I'll update the spec.